### PR TITLE
osd/osc: show appropriate icons when playing backwards

### PR DESCRIPTION
**Pull request by a-tze**
These commits enable OSC and OSD to show triangles pointing left when playback direction is set accordingly.

### CODE REVIEW

**Comment by wm4** [comment]
Does anyone know how we check whether the .otf was "properly" changed in cases like this?

**Comment by Akemi** [comment]
maybe @wiiaboo or @philipl have an idea.

otherwise checking if using gen-osd-font.sh results in the same otf file, i guess?

**Comment by a-tze** [comment]
Executing the script in the CI workflow and comparing the results sounds very good, though I think that there is sort of a timestamp in the otf file. So there might be a "reproducible build" problem. Not sure if the timestamp is just "now" or if it is taken from the font-property file.

**Comment by qmega** [comment]
FontForge provides `sfddiff`, and also has a compare function in its GUI.
```
$ sfddiff --warn old.otf new.otf
<snip fontforge copyright/version>
Outline Glyphs
 Glyphs in <snip>/new.otf but not in <snip>/old.otf
  Glyph “uniE00D” missing from <snip>/old.otf
```
Exit code does **not** indicate whether they're different, though.

**Comment by philipl** [comment]
Should we gen the otf in the build and not check it in at all?

**Comment by wm4** [comment]
Would that make fontforge a build dependency?

**Comment by philipl** [comment]
Yeah, it would. Perhaps that's excessive, but then we're stuck with checking the font in.

**Review by kasper93 — APPROVED** [review]
Why not. Won't cause any problems.

File: player/lua/osc.lua [hunk 1]
```diff
@@ -1796,7 +1796,11 @@ function osc_init()
 
     ne.content = function ()
         if mp.get_property("pause") == "yes" then
-            return ("\238\132\129")
+            if mp.get_property("play-dir", "forward") ~= "backward" then
```
Comment (guidocella) [inline player/lua/osc.lua:1799]: This should be updated to `play-direction` to not log a warning.

**Comment by a-tze** [comment]
This PR is pretty dusty, should I try to rebase it to current master? Unsure what the linter complains about regarding the commit message, the error message is empty.

File: player/lua/osc.lua (hunk 1 above)
Comment (guidocella) [reply player/lua/osc.lua]: Could also be shortened to:

```lua
return mp.get_property("play-direction") == "forward"
       and "\238\132\129"
       or "\238\132\144"
```

I have no idea why some other osc.lua functions put parentheses here but they don't do anything.

**Comment by kasper93** [comment]
> Unsure what the linter complains about regarding the commit message, the error message is empty.

I don't know what has happened here. Why the GHA workflow is running on this PR, while it is not on the source branch. Apparently actions/checkout doesn't fetch full repo 
> Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set fetch-depth: 0 to fetch all history for all branches and tags.

but it still shouldn't break like that, but either way not our issue.

LGTM.

**Pull request merged**